src/scss/components/text-input/_index.scss

```diff
@@ -138,6 +138,7 @@ hx-text-control {
 
   > p.hxErrorText {
     color: $red-status-500;
+    color: $rax-component-input-error-border-bottom-color;
```

Comment:
Please update to use IE11 fallback pattern:

```scss
color: $rax-component-input-error-border-bottom-color;
color: var(--hxTextControl-hxErrorText, $rax-component-input-error-border-bottom-color);
```

Reply:
Changes have been done, Please review